packages/overlay/README.md

```diff
@@ -39,112 +41,17 @@ By leveraging the `trigger` attribute to pass an ID reference to another element
         </sp-dialog>
     </sp-popover>
 </sp-overlay>
+
+<!-- Opening an overlay via a hover interaction -->
+<sp-overlay trigger="trigger@hover" placement="bottom">
+    <sp-tooltip>
+        I'm a tooltip and I'm triggered by hovering over the button!
+    </sp-tooltip>
+</sp-overlay>
 ```
 
 When a `<sp-overlay>` element is opened, it will pass that state to its direct children elements as the property `open`, which it will set to `true`. Elements should react to this by initiating any transition between closed and open that they see fit. Similarly, `open` will be set to `false` when the `<sp-overlay>` element is closed.
 
-### Action bar
```

Comment:
I'd like to have this somewhere. Thoughts on a good location?

Reply:
If we wanna keep that then I guess the location is fine (before we talk about the API)...